docs/tutorials/digital-twin-simulation/awsim-tutorial.md

```diff
@@ -0,0 +1,77 @@
+# AWSIM Simulator Setup
+
+This instruction presents how to run Tier IVs AWSIM with Autoware.
+
+## Preparation
+
+- Install Autoware following the official instruciton : [link](https://autowarefoundation.github.io/autoware-documentation/main/installation/autoware/source-installation/)
+- Download and extract AWSIM binary : [link](https://drive.google.com/file/d/1jgB_A5cSso9dzcNffQYthe07GG07QLQh/view?usp=sharing)
+- Download and extract map folder : [link](https://drive.google.com/file/d/1vGFI0o0zQ-gRZYqKrPbnrtCN3c3-92Fy/view?usp=sharing)
+
+## Running steps
+
+### AWSIM Simulator
+
+The AWSIM Simulator can be run without any further preparation. To run the simulator:
+
+- open the terminal navigate to the simulation directory and run:
+
+        source /opt/ros/galactic/setup.bash
+        cd <PATH_TO_EXTRACTED_FOLDER_WITH_AWSIM>
+        ./AWSIM.x86_64
+
+The properly working simulator should have the lidar rays visible and the data topics should appear when discovering using `ros2 topic list`.
```

Comment:
My suggestion says `the expected data topics` but are there any data topics that are specific to AWSIM that would indicate that is running correctly? If so, then listing those topics would be helpful to anyone running the simulator who wanted to be sure that it was working correctly.

```suggestion
If the simulator is working properly, Lidar rays should be visible and the expected data topics should appear when `ros2 topic list` is run.
```

Reply:
Hmm, We have created the simulation compatibility tests in other issue. It is still in progress but in general it's the package that runs sets of tests with running simulator to evaluate if interface is correct. I was planning to add the instruction about these tests in the same folder, and then maybe link the compatibility tests here ?

Reply:
I would keep it for now like it is, and link to in to the instruction with compatibility tests/interfaces in next PR with instructions for compatibility tests.